hieradata_aws/common.yaml

```diff
@@ -1433,7 +1433,11 @@ monitoring::checks::cache::servers:
   - 'backend-redis-002'
 
 monitoring::checks::rds::servers:
+   # NOTE: content-data-api-postgresql-primary is overridden in
+   # modules/monitoring/manifests/checks/rds.pp. If it is removed from this
+   # list, the override will also need to be removed.
    - 'content-data-api-postgresql-primary'
+
```

Comment:
I wonder what will be the effect of this empty line?

Reply:
accoridng to yq, this is allowed

Reply:
Yeah, I'm pretty sure it has no effect on the semantics. I just wanted it there to make it clear which line the comment referred to.